app/services/idv/analytics_events_enhancer.rb

```diff
@@ -50,7 +50,28 @@ def self.included(_mod)
     def common_analytics_attributes
       {
         proofing_components: proofing_components,
-      }
+        active_profile_idv_level: active_profile&.idv_level,
```

Comment:
Should these be in the IdV events enhancer or is there value in logging these on all analytics events?